homework-submission/server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
```

Comment:
Since you've made an express server based application, Where is your `package.json`?